feat(terra): better subrepo docs (#231)
FyraLabs/devdocs: pages/terra/installing.mdx

sudo dnf install --nogpgcheck --repofrompath 'terra,https://repos.fyralabs.com/terrael$releasever' terra-release
```

## Subrepos

<Callout type="info">

>     Subrepos are only available for `f*` branches (not `el*`).

Reviewer: ```suggestion
    Subrepos are not available on Enterprise Linux.
```

</Callout>

Terra provides additional packages in separate repositories. These packages might conflict with RPM Fusion or the Fedora repositories. Extra care and technical knowledge may be needed.

The current list of subrepos are `extras`, `nvidia`, `mesa`, `multimedia` (f43+ only).
They can be enabled by installing `terra-release-extras`, `terra-release-nvidia`, etc.

### Extras

> The extras subrepo provides packages that conflict with Fedora but do not belong to the rest of the subrepos.

Reviewer: ```suggestion
The extras subrepo provides miscellaneous packages that conflict with Fedora. Things like patched WINE and Switcheroo control are in Extras.
```

### Mesa

> In the Mesa subrepo, the `mesa` package is provided. Unlike Fedora and RPM Fusion, this build of mesa is complete, coherent, and with most if not all features enabled. This also contains patches from Universal Blue.

Reviewer: ```suggestion
The Mesa subrepo provides our build of Mesa, with patches from our friends at Universal Blue and most features enabled, unlike RPMFusion and Fedora's builds.
```

### NVIDIA

> The NVIDIA subrepo provides a different set of NVIDIA drivers based on the [Negativo17](https://negativo17.org/) builds.

Reviewer: ```suggestion
The NVIDIA subrepo provides a selection of drivers and software for NVIDIA GPUs. We offer drivers based on [Negativo17](https://negativo17.org/) builds, and CUDA packages.
```

### Multimedia

> The multimedia subrepo is only available to Fedora 43 and newer releases. Multimedia-related packages, including `gstreamer1` packages, `ffmpeg`, and other codecs are shipped in this subrepo.

Reviewer: ```suggestion
The multimedia subrepo is only available on Fedora 43 and newer. It provides multimedia codecs and related software, like `gstreamer1` packages and `ffmpeg`.
```
